SDAccel/userspace/src/perfmon_parameters.h

```diff
@@ -1,6 +1,36 @@
+/**
+ * Copyright (C) 2016-2017 Xilinx, Inc
```

Comment:
We have two headers here, so might have to revisit this later with Xilinx about keeping the correct one or unifying their headers